doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1122,10 +1122,10 @@ module can be accessed using the dot notation:
 
 .. _example_filter_module:
 
-.. example Creating a new module that omits some items from an existing module
+.. example:: Creating a new module that omits some items from an existing module
 
-   Since the type, :n:`SIG`, of the new module :n:`N` doesn't define y or give the body of :n`x`,
-   those are not included in :n:`N`.
+   Since :n:`SIG`, the type of the new module :n:`N`, doesn't define :n:`y` or
+   give the body of :n:`x`, which are not included in :n:`N`.
```

Comment:
There's an inconsistency between here and the use of `:g:` below. At some point, we should clarify what kind of verbatim quoting we want to keep and in which case we should use which.